examples/platform/silabs/SiWx917/matter_config.cpp

```diff
@@ -56,6 +56,10 @@ using namespace ::chip::DeviceLayer;
 
 #include "SilabsDeviceDataProvider.h"
 
+#ifdef CHIP_DEVICE_CONFIG_ICD_SERVER_ENABLE
+ICDManager SilabsMatterConfig::mICDManager;
```

Comment:
can it be in a common place for both silabs platforms

Reply:
Yes but in a follow refactor when the files are merged.